packages/aws-lambda/src/ssm.js

```diff
@@ -0,0 +1,143 @@
+/*
+ * (c) Copyright IBM Corp. 2022
+ * (c) Copyright Instana Inc. and contributors 2022
+ */
+
+'use strict';
+
+const ENV_NAME = 'INSTANA_SSM_PARAM_NAME';
```

Comment:
Is there a specific need to make the parameter name configurable? Or could we also just always fetch the same parameter name and tell users which parameter they need to set?

Reply:
Good question :) 

Two reasons why I think we want to leave it as it is

1. Naming patterns. If you add a new value to the parameter store you need to choose a name.
I can imagine that companies use naming patterns such as `/company/lambda/x` 

2. Visibility. Setting INSTANA_SSM_PARAM_NAME in their env shows them that they explicitly use SSM over the default way. 

I am happy to discuss that topic on a call if you think we want to define a name for the customers :)

Reply:
OTOH, maybe letting users configure the key might be more user friendly. TBH, I am not familiar enough with SSM to know how users would expect this to work (configurable or fixed key). Feel free to ignore this comment.

Reply:
> I am happy to discuss that topic on a call if you think we want to define a name for the customers :)

No, I think that's fine. We would still want an env var anyway to enable SSM usage (something like `INSTANA_USE_SSM=true`) and if keys can have a structure as you pointed out, letting users configure it absolutely makes sense. 👍